zng/docs/zeek-compat.md

```diff
@@ -1,138 +1,188 @@
 # ZNG Compatibility with Zeek Logs
 
-THIS SECTION IS A WORK IN PROGRESS.  WE DECIDED TO FOREGO COMPLETE
-FORMAT-COMPATIBILITY BETWEEN ZNG AND ZEEK AND WILL INSTEAD DOCUMENT HERE THE
-RELATIONSHIP BETWEEN ZEEK AND ZNG FORMAT.
+- [Introduction](#introduction)
+- [Equivalent Types](#equivalent-types)
+- [Example](#example)
+- [Type-Specific Details](#type-specific-details)
+  * [`double`](#double)
+  * [`set`](#set)
+  * [`enum`](#enum)
+  * [`string`](#string)
+  * [`record`](#record)
```

Comment:
Nit: I think alphabetical order is nicer than arbitrary order.

Reply:
@nwt: Hmm. FWIW, the order I chose wasn't wholly arbitrary, though I would accept it wasn't at all intuitive. [ 😄  ] I was effectively getting the easy/cosmetic ones out of the way first, then building my way up to the hairier ones. I doubt that was benefitting the reader in any way, so I'm open to changing it.

I'm not sure if this was obvious, but the _top_ table was intentionally put in the same order as the [Zeek Data Types](https://docs.zeek.org/en/current/script-reference/types.html) I was referencing. If that's seen as a reasonable thing to do, it seems I could extend that logic and make my "Additional Detail" sections follow the same order. Or... alphabetical, as you suggested.

Since you framed it as a nit, I know I'm probably already spending more time thinking about this as necessary. But you've opened my mind to changing the order, so now I'd be curious if you feel strongly enough about alphabetical that you'd be in favor of changing the order of the _top_ table as well, or if you'd think it's a-ok for me to change the Additional Details section to mirror the order of the top table.

Reply:
I've opted for re-order the entries in the "Additional Detail" section to match the order they appear in the top table. Happy to revisit in a future pass if you'd like.